src/main/java/com/exasol/releasedroid/adapter/github/GitHubRelease.java

```diff
@@ -63,6 +65,15 @@ public String getRepositoryName() {
         return this.repositoryName;
     }
 
+    /**
+     * Check if upload assets required.
+     *
+     * @return true if upload assets required
+     */
+    public boolean uploadAssets() {
```

Comment:
Maybe it is good idea to prefix this with `is` or `has`, it is clearer for boolean checks.

Reply:
Good idea, done!